Cargo.toml

```diff
@@ -12,5 +12,5 @@ rpds = "1.1.0"
 archery = "1.2.0"
 
 [dependencies.pyo3]
-version = "0.20.3"
+version = "0.22.0"
```

Comment:
[PyO3 0.22.1](https://github.com/PyO3/pyo3/releases/tag/v0.22.1) is out, so might as well bump.

Reply:
Thanks for the reminder! Just updated it!